### ISSUE TITLE
ut2k4server update fails: No appmanifest_.acf found.

### ISSUE THREAD

**Issue by ngallaher** [GameServerManagers/LinuxGSM#3008]
## User Story

As an LGSM server administrator, I attempted to install and then update a ut2k4 server using the framework.
The installation for all appearances has succeeded, and UT2k4 clients can connect and play.  However, upon executing `./ut2k4server update` I am presented with 

```
[ ERROR ] Updating ut2k4server: No appmanifest_.acf found
[ INFO ] Updating ut2k4server: Forcing update to correct issue
[ START ] Updating ut2k4server: SteamCMD
couldn't execute "+login": no such file or directory
    while executing
"spawn -noecho +login {} {} +force_install_dir /home/ut2k4server/serverfiles +app_update {} +quit"
    ("eval" body line 1)
    invoked from within
"eval [list spawn -noecho] $argv"
    invoked from within
"if {[string compare [lindex $argv 0] "-p"] == 0} {
    # pipeline
    set stty_init "-echo"
    eval [list spawn -noecho] [lrange $argv 1 end]
    clo..."
    (file "/usr/bin/unbuffer" line 13)
Complete! Updating ut2k4server: SteamCMD
[ FAIL ] Updating ut2k4server: Still no appmanifest_.acf found
```
## Basic info

* **Distro:** Ubuntu 18.04.5 LTS
* **Game:** Unreal Tournament 2004
* **Command:** update
* **LinuxGSM version:** v20.4.1

## Further Information

A clear description of what the bug is and any ideas on how to resolve it. Plus any further context that might be relevant to the issue.

## To Reproduce

Steps to reproduce the behaviour:

1. `linuxgsm.sh ut2k4server`
1. `./ut2k4server install`
1. Hit enter to not register the server with the pool
1. `./ut2k4server update`

## Expected behaviour

Update succeeds.

As far as I can tell, the problem is due to `steamcmdcommand` not being set to anything, but given that the application is not nominally a steam game, I'm not sure why it needs steamcmd.

**Comment by dgibbs64** [comment]
The option to update should not be available.

**Comment by h3o66** [comment]
This seems to be fixed in develop and should be fixed in the next relase

This was fixed for the next version in commit: 33eb01b387e36067471171655aeed939fd1df671
